### ISSUE TITLE
aquatone not scanning due to empty urllist.txt

### ISSUE THREAD

**Issue by shabbadont** [nahamsec/lazyrecon#51]
This happens no matter the domain I scan.

Recon started on xxx.com 
Listing subdomains using sublister...
Checking certspotter...
Checking http://crt.sh
Starting Massdns Subdomain discovery this may take a while
Massdns finished...
Started dns records check...
Looking into CNAME Records...
Excluding domains (if you set them with -e)...
Subdomains that have been excluded from discovery:

Starting discovery...
Probing for live hosts...
cat: ./xxx/recon-2021-02-05/urllist.txt: No such file or directory
Total of 1 live subdomains were found
Starting aquatone scan...
No targets found in input.
Scraping wayback for data...
Wordlist saved to /xxxx/recon-2021-02-05/wayback-data/paramlist.txt
Starting dirsearch...
wc: /root/tools/dirsearch/reports//: Is a directory
Host discovery disabled (-Pn). All addresses will be marked 'up' and scan times will be slower.
Scan for xxx.com finished successfully
Scan completed in : 244 minutes and 45 seconds.

**Comment by thedecoder12** [comment]
@incredincomp  messed up the script.

**Comment by incredincomp** [comment]
Yeah, that's true.. thanks. This pull request should fix it. Simple check was needed and I spaced it

**Comment by shabbadont** [comment]
Thank you very much

**Comment by incredincomp** [comment]
Yeah bud, I'm sorry 😓 it was a stupid oversight on my part. Trying to get it merged now but you can totally just copy paste the new function or use my fork
